test/Npgsql.Tests/PgPassFileTests.cs

```diff
@@ -11,27 +11,27 @@ public class PgPassFileTests
         public void ShouldParseAllEntries()
         {
             var file = PgPassFile.Load(_pgpassFile);
-            var entries = file.Entries.ToList();
+            var entries = file!.Entries.ToList();
```

Comment:
I might be wrong and it certainly is a matter of taste but I'd rather put the bang behind `PgPassFile.Load(_pgpassFile)` which would translate to: "I know that in this special case `PgPassFile.Load(_pgpassFile)` will not return null." which is a very clear description of the behavior.
The current way translates to: "I know that in this case `file` isn't null", which is also true but less clear IMO.

Reply:
I agree, it's better your way.